### PR TITLE
Rework targeted webview debug code to work after about:blank change

### PR DESCRIPTION
**Pull request by DavidShoe**
Changed code to now watch for navigation events and look for the targeted url. Once found we activate the debugging mode over the correct port.

Also some lint cleanups

### CODE REVIEW

File: package.nls.cs.json [hunk 1]
```diff
@@ -1,33 +1,35 @@
 {
-	"extension.description": "Debug your JavaScript code in the Edge browser, or any other target that supports the Edge Debugger protocol.",
```
Comment (EricCornelson) [inline package.nls.cs.json:2]: Where did these translations come from?

If we have missing translations, I can reach out to our localization lead. I'm not sure these nls files are actually used in the final packaged build

**Comment by EricCornelson** [comment]
Also, @DavidShoe, I'm having trouble testing these changes locally. Using the WebView2APISample code, I'm getting the following error:

image

Any insight on what might be the issue?

**Comment by DavidShoe** [comment]
I think they were auto generated, when I did a “gulp build”, the loc files changed I suspected that it isn’t the right way to update them (especially with the warnings about missing strings).

I can revert those changes.

File: package.nls.cs.json (hunk 1 above)
Comment (DavidShoe) [reply package.nls.cs.json]: Removed the autogen loc chagnes.

**Pull request merged**